### DISCUSSION TITLE
add "squircle" className as alternative to rounded-full

### DISCUSSION THREAD

**Discussion by PeerRich** [tailwindlabs/tailwindcss#10493] [Ideas]
![CleanShot 2023-02-05 at 17 58 49@2x](https://user-images.githubusercontent.com/8019099/216833202-af9e5e94-9593-4ce0-8577-86f9cadab126.jpg)

https://en.wikipedia.org/wiki/Squircle

alternatives:

https://github.com/pie6k/react-ios-corners/

this can be achieved with css mask

**Comment by PeerRich** [comment]
may be out of scope as it requires a bit more JS than i expected. I thought this was possible with pure CSS. feel free to close

**Comment by PeerRich** [comment]
looks like daisy ui found a css based solution https://daisyui.com/components/avatar/#avatar-with-mask

**Comment by KlotzJesse** [comment]
@PeerRich Is now native in CSS, https://www.npmjs.com/package/tailwindcss-corner-shape or https://github.com/[toolwind/corner-shape](https://github.com/toolwind/corner-shape)

**Reply by PeerRich** [reply]
siuuu
